### DISCUSSION TITLE
Automatically fetch sd-concepts-library textual embeddings

### DISCUSSION THREAD

**Discussion by GabrielVidal1** [civitai/civitai#27] [Ideas]
A lot of website just like civit.ai automatically fetch and display textual embeddings from the [sd-concepts-library](https://huggingface.co/sd-concepts-library) space on huggingface, there are almost 800 textuals embeddings available!

**Comment by JustMaier** [comment]
We've pulled in all of these textual inversions using our new HuggingFace importer.
The next improvement would be to make it automatically poll that library for new ones.
